### ISSUE TITLE
Carbon\Exceptions\UnknownUnitException with message 'Unknown unit 'jour'.'

### ISSUE THREAD

**Issue by labomatik** [kylekatarnls/enhanced-period#8]
With the current lib converting a CarbonPeriod to Spatie Period is throwing an error

```
Carbon\Exceptions\UnknownUnitException with message 'Unknown unit 'jour'.'
```

Do you have any idea?

**Comment by labomatik** [comment]
Using App::setLocale('en'); is fixing the issue

**Comment by kylekatarnls** [comment]
Hello,

"jour" is the french word for "day", so a method received a translated input while expecting an english string. And this exception is expected to occur when such input is given. I can't say more without the stack trace to know from which method it was thrown and a code snippet to reproduce the error.

Please also include the versions of:
- PHP
- enhanced-period
- Carbon
- framework (such as Laravel if used)

So I can reproduce using the same setup.

Thanks.

**Comment by labomatik** [comment]
Hello Kyle, sorry, i forgot to add more infos:
It's a plain laravel with locale in FR (App::setLocale('fr'))
PHP 8.0
Enhanced-period: 1.2
Carbon: Carbon 2
Laravel Framework: 8.69

Snippet (same as in doc):
```
use Carbon\CarbonPeriod;
use Cmixin\EnhancedPeriod;

CarbonPeriod::mixin(EnhancedPeriod::class); // This line is not needed if you use Laravel default auto-discovery.

// Use toEnhancedPeriod to convert `Carbon\CarbonPeriod` objects to `Spatie\Period\Period` ones
$a = CarbonPeriod::create('2018-01-01', '2018-01-10')->toEnhancedPeriod();
$b = CarbonPeriod::create('2018-01-15', '2018-01-31')->toEnhancedPeriod();
```

**Comment by kylekatarnls** [comment]
OK, I setup a Laravel 8 app in French and reproduced. Working on.

**Comment by kylekatarnls** [comment]
Fixed with 2.57.0 release of Carbon you can `composer update` (or `composer update nesbot/carbon` to update only specifically carbon) to get the fix.
